tensorflow_datasets/core/dataset_builder.py

```diff
@@ -821,32 +877,38 @@ def builder_configs(cls):
     return config_dict
 
 
+@six.add_metaclass(abc.ABCMeta)
```

Comment:
Please only add typing annotations. Do not try to modify the logic on the code. The meta-class is already added by inheritance with `DatasetBuilder`

Reply:
Done